doc/configtables/sector.csv

```diff
@@ -63,6 +63,7 @@ tes_tau,,,The time constant used to calculate the decay of thermal energy in the
 -- decentral,days,float,The time constant in decentralized thermal energy storage (TES)
 -- central,days,float,The time constant in centralized thermal energy storage (TES)
 boilers,--,"{true, false}",Add option for transforming electricity into heat using resistive heater
+resistive_heaters,--,"{true, false}",Add option for using resistive heaters separately from boilers

```

Comment:
```suggestion
resistive_heaters,--,"{true, false}",Add option for transforming electricity into heat using resistive heaters (independently from gas boilers)

```

Reply:
Thanks for the notes, I have addressed them.